WordPress/gutenberg
# [Data Views][Grid Layout] Making cmd/ctrl clicking more apparent

See also: #59188
Will need to be refactored a bit if #59279 lands first, but the concept will be the same.

## What?

Makes it clearer that <kbd>Cmd</kbd>/<kbd>Ctrl</kbd>-clicking on a grid item is an available action.


## Why?

It is not currently clear that grid items can be <kbd>Cmd</kbd>/<kbd>Ctrl</kbd>-clicked.


## How?

- Changes the grid item cursor when the <kbd>Cmd</kbd>/<kbd>Ctrl</kbd> key is pressed, rather than waiting for the click.

## Testing Instructions

- Moving the mouse pointer over a grid item shouldn't indicate any change
- Pressing and holding the <kbd>Cmd</kbd>/<kbd>Ctrl</kbd> key when the pointer is over a grid item should change the cursor to a `➕`
- Pressing and holding the <kbd>Cmd</kbd>/<kbd>Ctrl</kbd> key then moving the pointer over a grid item should change the cursor to a `➕`
- Pressing and holding the <kbd>Cmd</kbd>/<kbd>Ctrl</kbd> key then clicking anywhere on a grid item should toggle its selected state. 

### Testing Instructions for Keyboard

This change does not impact keyboard interaction.

## Screenshot

![Grid interaction showing cursor/behavioural changes with the command key pressed](https://github.com/WordPress/gutenberg/assets/159848/8ac7a6be-fe08-4762-a9fc-03be422f2624)
